modules/gcp/centos-gfx/centos-gfx-startup.sh.tmpl

```diff
@@ -276,20 +275,20 @@ join_domain()
         exitCode=$?
         if [[ $exitCode -eq 0 ]]
         then
-            log "--> Joined Domain '${domain_name}' and OU '$OU'"
+            log "--> Joined Domain '${domain_name}' and OU '$OU'."
```

Comment:
OU seems to be undefined - then nothing gets displayed here. don't need to display the OU unless you put in a check to see if OU exist first like the above if [[ -n "$OU" ]]